docs/release/trg-7/trg-7-09.md

```diff
@@ -0,0 +1,144 @@
+
+In case of repositories belonging to a [Product](https://eclipse-tractusx.github.io/community/products):
+2. The product of the repository has not been released within the *past year's major release cycle* **\***.
+3. The standards used by the product have been changed in a major version within the *past year's major release cycle* **\***.
+
+**\*** The *past year's major release cycle* is the time span between the breaking change releases YY.06 e.g., 24.05 till 24.05.
+
+### Deprecation Process
+
+To ensure the communities awareness the following process is used at least per major release planning:
+
+```mermaid
+graph TD
+    A[1: Prepare list of unmaintained repositories] --> B[2-1: Draft critical issue for committer meeting]
+    B --> C[2-2: Inform Eclipse Tractus-X community]
+    C --> D[3: 2 week period: Comment on issue for criteria discussion]
+    D --> E[4-1: Committer meeting: Discuss list]
+    E --> F[4-2: Send email to mailing list]
+    F --> G[5: Open Planning: Communicate deprecation proposal during Open Planning]
+    G --> H{6: Action taken?}
+    H -->|No| I[End of Release: Deprecate repository]
+    H -->|Yes| J[Plan Update of repository]
+    J --> K[Create issue in .eclipsefdn]
+```
+
+1. The committers prepare a list of unmaintained product repositories (**criteria 1**).
+2. The committers inform the Eclipse Tractus-X community via the [mailing list](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute/#dev-mailinglist). This is done by an issue with checkboxes listing all these repositories (find the issue template [below](#issue-template-for-deprecation-list)). Additionally a critical issue is drafted on the [board for the committer meeting](https://github.com/orgs/eclipse-tractusx/projects/61/views/6) in at least 2 full weeks.
+3. 2 weeks period: The committers comment the issue regarding which repositories are subject to **criteria 1** and **criteria 2**. Also if **Criteria 2** does not apply, deprecation may still be possible due to security concerns.
+4. Committer meeting: The committers discuss the list and check the boxes in the issue. A Mail is sent out to the Eclipse Tractus-X mailing list. This list is splitted into platform products (e.g. Tractus-X EDC) and use case products (e.g. PURIS). See a template below.
+5. During the [Open Planning](https://catenax-ev.github.io/docs/next/working-model/release-management/planning/tx-release-planning-days) the deprecation proposal is communicated again.
+6. If no action is taken, during the Release of the major version, the project leads will decide for the deprecation of the given repository and communicate accordingly with the Catena-X Association's release management or the specific organization responsible for the repository.
+7. In case the repository got deprecated, and there is again maintainace or develop capacity, the following procedure **MUST** be followed [Repository Reactivation Process] by a committer.
```

Comment:
```suggestion
5. If no action is taken within additional **4 weeks** the project leads will proceed with the deprecation of the given repositories and communicate accordingly on the dev mailing list.
6. In case the repository got deprecated, and there is again maintenance or develop capacity, the following procedure **MUST** be followed [Repository Reactivation Process](repository-reactivation-process) by a committer.
```

Reply:
done